other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -46,13 +46,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    materials = list(selected_interface)
</pre><p>We should not need <code>list</code> here, just return the list for  the <code>get_interfaces_...</code> function </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>We probably don't need print statement here either. We should instead add a visualization function to plot the selected interface(s) in a subsequent task</p>